core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -202,8 +200,8 @@ public synchronized void connect() throws IOException, ConnectionFailedException
         }
         throw e;
       } catch (TTransportException e) {
-        LOG.error("Failed to connect (" + retry.getRetryCount() + ") to " + getServiceName() + " "
-            + mMode + " @ " + mAddress + " : " + e.getMessage());
+        LOG.warn("Failed to connect ({}) to {} {} @ {}: {}", retryPolicy.getRetryCount(),
+            getServiceName(), mMode, mAddress, e.getMessage());
```

Comment:
I think for anything that will get retried we want just `e.getMessage()` to limit the verbosity